# VRE_AMR-EZ_2025
----------------------
References and higher-quality plots for OH: AMR & EZ poster presentation (March 2025)

Ashton N Sies (ans227@uregina.ca)

### Works cited
----------------------
##### 1: Islam, M., Sharon, B., Abaragu, A., Sistu, H., Akins, R. L., & Palmer, K. (2023). Vancomycin Resistance in Enterococcus faecium from the Dallas, Texas, Area Is Conferred Predominantly on pRUM-Like Plasmids. mSphere, 8(2), e0002423. https://doi.org/10.1128/msphere.00024-23

##### 2: Wick, R. R., Judd, L. M., Gorrie, C. L., & Holt, K. E. (2017). Unicycler: Resolving bacterial genome assemblies from short and long sequencing reads. PLoS computational biology, 13(6), e1005595. https://doi.org/10.1371/journal.pcbi.1005595

##### 3: Wick, R. R., Judd, L. M., Cerdeira, L. T., Hawkey, J., Méric, G., Vezina, B., Wyres, K. L., & Holt, K. E. (2021). Trycycler: consensus long-read assemblies for bacterial genomes. Genome biology, 22(1), 266. https://doi.org/10.1186/s13059-021-02483-z 

##### 4: Oxford Nanopore Technologies. Medaka. https://github.com/nanoporetech/medaka

##### 5: Wick, R. R., & Holt, K. E. (2022). Polypolish: Short-read polishing of long-read bacterial genome assemblies. PLoS computational biology, 18(1), e1009802. https://doi.org/10.1371/journal.pcbi.1009802

##### 6: Zimin, A. V., & Salzberg, S. L. (2020). The genome polishing tool POLCA makes fast and accurate corrections in genome assemblies. PLoS computational biology, 16(6), e1007981. https://doi.org/10.1371/journal.pcbi.1007981

##### 7: Jolley, K. A., Bray, J. E., & Maiden, M. C. J. (2018). Open-access bacterial population genomics: BIGSdb software, the PubMLST.org website and their applications. Wellcome open research, 3, 124. https://doi.org/10.12688/wellcomeopenres.14826.1

##### 8: Petkau, A., Mabon, P., Sieffert, C., Knox, N. C., Cabral, J., Iskander, M., Iskander, M., Weedmark, K., Zaheer, R., Katz, L. S., Nadon, C., Reimer, A., Taboada, E., Beiko, R. G., Hsiao, W., Brinkman, F., Graham, M., & Van Domselaar, G. (2017). SNVPhyl: a single nucleotide variant phylogenomics pipeline for microbial genomic epidemiology. Microbial genomics, 3(6), e000116. https://doi.org/10.1099/mgen.0.000116

##### 9: Schwengers, O., Jelonek, L., Dieckmann, M. A., Beyvers, S., Blom, J., & Goesmann, A. (2021). Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification. Microbial genomics, 7(11), 000685. https://doi.org/10.1099/mgen.0.000685

##### 10: Wishart, D. S., Han, S., Saha, S., Oler, E., Peters, H., Grant, J. R., Stothard, P., & Gautam, V. (2023). PHASTEST: faster than PHASTER, better than PHAST. Nucleic acids research, 51(W1), W443–W450. https://doi.org/10.1093/nar/gkad382

##### 11: Alcock, B. P., Huynh, W., Chalil, R., Smith, K. W., Raphenya, A. R., Wlodarski, M. A., Edalatmand, A., Petkau, A., Syed, S. A., Tsang, K. K., Baker, S. J. C., Dave, M., McCarthy, M. C., Mukiri, K. M., Nasir, J. A., Golbon, B., Imtiaz, H., Jiang, X., Kaur, K., Kwong, M., … McArthur, A. G. (2023). CARD 2023: expanded curation, support for machine learning, and resistome prediction at the Comprehensive Antibiotic Resistance Database. Nucleic acids research, 51(D1), D690–D699. https://doi.org/10.1093/nar/gkac920

##### 12: Carattoli, A., Zankari, E., García-Fernández, A., Voldby Larsen, M., Lund, O., Villa, L., Møller Aarestrup, F., & Hasman, H. (2014). In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing. Antimicrobial agents and chemotherapy, 58(7), 3895–3903. https://doi.org/10.1128/AAC.02412-14

##### 13: Redondo-Salvo, S., Bartomeus-Peñalver, R., Vielva, L., Tagg, K. A., Webb, H. E., Fernández-López, R., & de la Cruz, F. (2021). COPLA, a taxonomic classifier of plasmids. BMC bioinformatics, 22(1), 390. https://doi.org/10.1186/s12859-021-04299-x 
